components/report-details/support/processing-drawer.tsx

```diff
@@ -0,0 +1,38 @@
+import { Loader } from "lucide-react";
+import { Drawer } from "vaul";
+import { Button } from "../../ui/button";
+
+interface SupportProcessingDrawerProps {
+	container: HTMLDivElement | null;
+}
+
+export default function SupportProcessingDrawer({
+	container,
+}: SupportProcessingDrawerProps) {
+	return (
+		<Drawer.Root dismissible={false}>
+			<Drawer.Trigger asChild>
+				<Button size={"lg"} variant={"default"} type="submit">
+					Support this report
+				</Button>
+			</Drawer.Trigger>
+			<Drawer.Portal container={container}>
+				<Drawer.Overlay className="fixed inset-0 bg-stone-900/60" />
+				<Drawer.Content className="bg-stone-100 flex flex-col rounded-t-xl max-h-[50%] fixed bottom-0 left-0 right-0 after:hidden">
+					<div className="flex flex-col gap-4 px-4 py-6">
+						<div className="flex justify-center animate-spin">
+							<Loader />
```

Comment:
Using Loader2 everywhere else.
```suggestion
							<Loader2 />
```